### PR TITLE
fixed issue #22

### PR DESCRIPTION
**Pull request by kasorin**
- change frame_effect to frame_effects
- add preview

### CODE REVIEW

File: src/tests/Main.ts [hunk 1]
```diff
@@ -266,6 +266,33 @@ describe("Scry", function () {
...
+				expect(card.preview.source_uri).eq("https://www.cracked.com/article_27000_a-first-look-at-unsanctioned-magic-gatherings-new-set.html");
+				expect(Scry.error()).eq(undefined);
+			});			
+			
+			it("preview.source", async () => {
+				const card = await Scry.Cards.byId("12a035fe-8847-4678-84f7-01bac77ae011");
+				expect(card.preview.source).eq("Cracked");
+				expect(Scry.error()).eq(undefined);
+			});
+		});
```
Comment (Yuudaari) [inline src/tests/Main.ts]: This is testing Scryfall, not the Scryfall SDK. Should be removed

File: src/api/Cards.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ enum Rarity {
 	mythic,
 }
 
-enum FrameEffect {
+enum FrameEffects {
```
Comment (Yuudaari) [inline src/api/Cards.ts]: Use the old name

**Comment by kasorin** [comment]
Thank you! I'll fix it.

**Comment by Yuudaari** [comment]
Looks good, thanks again! I'll publish a new version when I get a sec 🤍

**Pull request merged**